package.json

```diff
@@ -21,7 +21,7 @@
     "test": "mocha"
   },
   "dependencies": {
-    "node-sass": ">=1.0.1 <3.0.0",
```

Comment:
The version should not be open-ended - future major bumps can break compatibility. Also the range should include all the versions the plugin is compatible with, giving users the ability to stick with an older one if they need.

`>=1.0.1 <4.0.0` would probably be appropriate if you've confirmed it's working with 3.x